tests/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+project(${CMAKE_PROJECT_NAME}Tests CXX)
+
+include(CTest)
```

Comment:
include(CTest) and if(BUILD_TESTING) should likely be in the main CMakeLists.txt, before the ``add_subdirectory(tests)``

Reply:
You mean like https://github.com/OSGeo/shapelib/pull/113/commits/3df250b193750a1167cdd1b5f62d2708d56d2587? Not sure what is the best practice.